packages/cli/src/commands/governance/test-proposal.ts

```diff
@@ -0,0 +1,52 @@
+import { toTxResult } from '@celo/connect'
+import { ProposalBuilder, proposalToJSON, ProposalTransactionJSON } from '@celo/governance'
+import { flags } from '@oclif/command'
+import { readFileSync } from 'fs'
+import { BaseCommand } from '../../base'
+import { printValueMapRecursive } from '../../utils/cli'
+import { Flags } from '../../utils/command'
+export default class Propose extends BaseCommand {
```

Comment:
this should be renamed and probably made hidden
it seems dangerous for non-developers to use this?